api/src/main/java/org/pmiops/workbench/reporting/ReportingMapper.java

```diff
@@ -3,28 +3,23 @@
 import java.util.Collection;
 import java.util.List;
 import org.mapstruct.Mapper;
-import org.mapstruct.Mapping;
-import org.pmiops.workbench.db.model.DbUser;
-import org.pmiops.workbench.db.model.DbWorkspace;
-import org.pmiops.workbench.model.ReportingResearcher;
-import org.pmiops.workbench.model.ReportingWorkspace;
+import org.pmiops.workbench.db.dao.projection.PrjUser;
+import org.pmiops.workbench.db.dao.projection.PrjWorkspace;
+import org.pmiops.workbench.db.model.DbStorageEnums;
+import org.pmiops.workbench.model.BqDtoUser;
+import org.pmiops.workbench.model.BqDtoWorkspace;
 import org.pmiops.workbench.utils.mappers.CommonMappers;
 import org.pmiops.workbench.utils.mappers.MapStructConfig;
 
 @Mapper(
     config = MapStructConfig.class,
-    uses = {CommonMappers.class})
+    uses = {CommonMappers.class, DbStorageEnums.class})
```

Comment:
Static methods are fair game for MapStruct, so all I have to do to get enum maps is include `DbStorageEnums`